README.md

```diff
@@ -205,69 +205,56 @@ docker run --rm --name pg1 -e PGSO_INSTANCE_NAME=pg1 -e PGSO_REGION=eu-north-1 \
   pgspotops/pg-spot-operator:latest
 ```
 
-## Cleanup of all operator created objects
-
-After some work/testing one can clean up all operator created cloud resources via `PGSO_TEARDOWN_REGION` or only a
-single instance via the `PGSO_TEARDOWN` flag.
+# Security
 
-```
-docker run --rm --name pg1 -e PGSO_TEARDOWN_REGION=y -e PGSO_REGION=eu-north-1 \
-  -e PGSO_AWS_ACCESS_KEY_ID="$(grep -m1 aws_access_key_id ~/.aws/credentials | sed 's/aws_access_key_id = //')" \
-  -e PGSO_AWS_SECRET_ACCESS_KEY="$(grep -m1 aws_secret_access_key ~/.aws/credentials | sed 's/aws_secret_access_key = //')" \
-  pgspotops/pg-spot-operator:latest
-```
+By default, security is not super trimmed down, as the experience is geared towards more casual or temporary workloads.
+Everything is tunable though.
 
-or by running a helper script from the "scripts" folders (by default just lists the resources, need to add a parameter):
+**ATTENTION** For all public Spot Operator managed instances it is very much **recommended to create a new special
+purpose VPC**, isolated from the rest of the account!
 
-```
-./scripts/aws/delete_all_operator_tagged_objects.sh yes
-```
-
-A third option for local Docker convenience, to avoid a restart with different env. inputs, is to create a dummy "destroy
-file" in the container that signals instance shutdown:
+Especially as this is as simple as running a `terraform apply` for
+example from [here](https://github.com/terraform-aws-modules/terraform-aws-vpc/tree/master/examples/simple) - just replace
+*private_subnets* with *public_subnets*, and your region of choice in *locals.region*. This concrete Terraform example
+doesn't add any Security group rules though, so that you might want to add some SG rules like below for the freshly created VPC
+(tuning the CIDR according to your needs / location).
 
 ```commandline
-docker run ...
-# Note the destroy file path from the log
-2024-09-30 10:49:38,973 INFO Instance destroy signal file path: /tmp/destroy-pg1
-...
-docker exec -it pg1 touch /tmp/destroy-pg1
-# On the next loop the resources will be cleaned up and the container shuts down
+SG_ID=mysgid
+REGION=myregion
+PORTS_TO_OPEN="22 5432 3000"  # See below for a description of all used ports
+
+for x in $PORTS_TO_OPEN ; do ;
+  aws ec2 authorize-security-group-ingress \
+      --group-id $SG_ID \
+      --protocol tcp \
+      --port $x \
+      --cidr "0.0.0.0/0" \
+      --region $REGION
+done
+
+aws ec2 authorize-security-group-egress \
+    --group-id $SG_ID \
+    --protocol "-1" \
+    --cidr "0.0.0.0/0" \
+    --region $REGION
 ```
 
-PS The operator tags all created object with special `pg-spot-operator-\*` tags, thus uses those also for the cleanup.
+## Security-relevant attributes with defaults
 
-# Local non-cloud development
+When using Docker / CLI:
 
-For Postgres setup testing, manifest handling etc, one can get by with local Virtualbox/Vagrant VMs, by using
-according flags:
+* --public-ip (Defaults to true)
```

Comment:
we should really be using prefixes like `is/has/creates/uses/etc` e.g.

Reply:
I've went with `--assign-public-ip` which is a bit better. "is/has" doesn't suit too well in this context I think. AWS for example uses `--associate-public-ip-address`